catroidSourceTest/build.gradle

```diff
@@ -5,7 +5,6 @@ repositories{
 apply plugin: 'java'
 
 dependencies {
-    testCompile files( project(":").buildDir.getPath() + '/intermediates/classes/debug' )
```

Comment:
@aried3r , @xorrizon: Why do we need to compile the `/intermediates/classes/debug`directory?